eventparser/eventparser_test.go

```diff
@@ -0,0 +1,166 @@
+	  "pulls_url": "https://api.github.com/repos/ksputo/hack-showcase/pulls{/number}",
+	  "milestones_url": "https://api.github.com/repos/ksputo/hack-showcase/milestones{/number}",
+	  "notifications_url": "https://api.github.com/repos/ksputo/hack-showcase/notifications{?since,all,participating}",
+	  "labels_url": "https://api.github.com/repos/ksputo/hack-showcase/labels{/name}",
+	  "releases_url": "https://api.github.com/repos/ksputo/hack-showcase/releases{/id}",
+	  "deployments_url": "https://api.github.com/repos/ksputo/hack-showcase/deployments",
+	  "created_at": "2019-07-16T07:44:59Z",
+	  "updated_at": "2019-07-17T11:53:39Z",
+	  "pushed_at": "2019-07-04T12:44:24Z",
+	  "git_url": "git://github.com/ksputo/hack-showcase.git",
+	  "ssh_url": "git@github.com:ksputo/hack-showcase.git",
+	  "clone_url": "https://github.com/ksputo/hack-showcase.git",
+	  "svn_url": "https://github.com/ksputo/hack-showcase",
+	  "homepage": null,
+	  "size": 8,
+	  "stargazers_count": 0,
+	  "watchers_count": 0,
+	  "language": null,
+	  "has_issues": false,
+	  "has_projects": true,
+	  "has_downloads": true,
+	  "has_wiki": false,
+	  "has_pages": false,
+	  "forks_count": 0,
+	  "mirror_url": null,
+	  "archived": false,
+	  "disabled": false,
+	  "open_issues_count": 0,
+	  "license": {
+		"key": "apache-2.0",
+		"name": "Apache License 2.0",
+		"spdx_id": "Apache-2.0",
+		"url": "https://api.github.com/licenses/apache-2.0",
+		"node_id": "MDc6TGljZW5zZTI="
+	  },
+	  "forks": 0,
+	  "open_issues": 0,
+	  "watchers": 0,
+	  "default_branch": "master"
+	},
+	"sender": {
+	  "login": "ksputo",
+	  "id": 26189888,
+	  "node_id": "MDQ6VXNlcjI2MTg5ODg4",
+	  "avatar_url": "https://avatars2.githubusercontent.com/u/26189888?v=4",
+	  "gravatar_id": "",
+	  "url": "https://api.github.com/users/ksputo",
+	  "html_url": "https://github.com/ksputo",
+	  "followers_url": "https://api.github.com/users/ksputo/followers",
+	  "following_url": "https://api.github.com/users/ksputo/following{/other_user}",
+	  "gists_url": "https://api.github.com/users/ksputo/gists{/gist_id}",
+	  "starred_url": "https://api.github.com/users/ksputo/starred{/owner}{/repo}",
+	  "subscriptions_url": "https://api.github.com/users/ksputo/subscriptions",
+	  "organizations_url": "https://api.github.com/users/ksputo/orgs",
+	  "repos_url": "https://api.github.com/users/ksputo/repos",
+	  "events_url": "https://api.github.com/users/ksputo/events{/privacy}",
+	  "received_events_url": "https://api.github.com/users/ksputo/received_events",
+	  "type": "User",
+	  "site_admin": false
+	}
+  }	
+`)
+
+func TestGetEventRequestPayload(t *testing.T) {
+
+	t.Run("Should create expected payload when the function is called with correct arguments", func(t *testing.T) {
+		//given
+		eventType := "star"
+		eventTypeVersion := "v1"
+		eventID := ""
+
+		//when
+		p, _ := GetEventRequestPayload(eventType, eventTypeVersion, eventID, data)
```

Comment:
You should check for errors here, not ignore them:
```suggestion
		p, err := GetEventRequestPayload(eventType, eventTypeVersion, eventID, data)
		require.NoError(t, err)
```